.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Checklist
+
+- Is this feature complete or do we expect follow-up changes before this code can ship?
+  - [ ] Yes
+  - [ ] No
+- Are you including unit tests for the changes and scenario tests if relevant?
+  - [ ] Yes
+  - [ ] No
+- Did you add public API? Did you have an API Review for it?
+  - [ ] Yes
+  - [ ] No
+- Does the change make any security assumptions or guarantees? Have you done a threat model and had a security review?
+  - [ ] Yes
+  - [ ] No
```

Comment:
For these 2 lines, they both have 2 questions and the answer can be different for each question. How would you answer that this changes public API but doesn't have an API Review (yet)?

Reply:
Good feedback, I'll fix this.